include/exadg/incompressible_navier_stokes/spatial_discretization/operator_coupled.cpp

```diff
@@ -213,13 +215,9 @@ OperatorCoupled<dim, Number>::rhs_stokes_problem(BlockVectorType & dst, double c
 template<int dim, typename Number>
 void
 OperatorCoupled<dim, Number>::apply_linearized_problem(BlockVectorType &       dst,
-                                                       BlockVectorType const & src,
-                                                       double const &          time,
-                                                       double const & scaling_factor_mass) const
+                                                       BlockVectorType const & src) const
 {
   // (1,1) block of saddle point matrix
-  this->momentum_operator.set_time(time);
-  this->momentum_operator.set_scaling_factor_mass_operator(scaling_factor_mass);
```

Comment:
In the previous implementation, `scaling_factor_mass` is only set in `apply_linearized_problem(), which is the `vmult()` function of the Krylov solver. Hence, the preconditioner does not know about this factor.

Reply:
no, it is exactly the `momentum_operator` that needs it. However, when setting these parameters in the `vmult()` function, the preconditioner can not query this data from the `momentum_operator` during `preconditioner->update()`.

Reply:
... because `preconditioner->update()` is called before the code runs into `operator->vmult()` in the Krylov solver.